src/TaskListClass.js

```diff
@@ -0,0 +1,141 @@
+import TaskObject from './TaskObject.js';
+import completedIsFalse from './CompletedFilter.js';
+import Drag from './drag-handle-minor-svgrepo-com.svg';
+import Bin from './bin-svgrepo-com.svg';
+
+class TaskList {
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem('To-Do List')) || [];
+  }
+
+  init() {
+    const addButton = document.getElementById('add-btn');
+    addButton.addEventListener('click', (event) => {
+      event.preventDefault();
+      this.addingEventListener();
+    });
+
+    document.getElementById('add-task').addEventListener('keypress', (event) => {
+      if (event.key === 'Enter') {
+        event.preventDefault();
+        this.addingEventListener();
+      }
+    });
+
+    const clearBtn = document.getElementById('clear');
+    clearBtn.addEventListener('click', () => {
+      this.clearCompleted();
+    });
+
+    window.addEventListener('load', () => {
+      this.renderList();
+    });
+  }
```

Comment:
- Good use of best practices of JS...... like "const" instead of "let" or "var"

Reply:
Thank you @AbuTalha3 for this catch! I've changed the description :)